apps/index.html

```diff
@@ -305,6 +305,22 @@
       </div>
 
       <div class="app">
+        <button onclick="install('118d77fa72eba6a685cb493bffb18fae?url=http://github.com/libreboard/libreboard/releases/download/v0.7.1/libreboard.spk')">Install »</button>
```

Comment:
It would be nice if the `http://` github.com URL were instead a `https://` github.com URL.

That's mostly due to my personal preference for https URLs wherever possible.